karton/core/main.py

```diff
@@ -168,6 +179,9 @@ def main() -> None:
 
     subparsers.add_parser("list", help="List active karton binds")
 
+    logs_parser = subparsers.add_parser("logs", help="Start streaming logs")
+    logs_parser.add_argument("--filter", help="Service identity filter", required=False)
```

Comment:
Maybe a super :smol: example of such filter? e.g. `karton.classifier-*`?

Reply:
fixed